index.html

```diff
@@ -7,7 +7,82 @@
     <link rel="stylesheet" href="style.css">
     <title>Assignment 1</title>
 </head>
+
 <body>
-    <h1>comproDLS</h1>
+    <main>
+        <div class="container">
+            <div class="container__left">
+                <div class="info__area">
+                    <p class="info__area--h1">
+                        comproDLS<span>TM</span>
+                    </p>
+                    <p class="info__area--h2">
+                        OnBoarding
+                    </p>
+                    <hr id="info__area--line">
+                    <p class="info__area--h3">
+                        Application
+                    </p>
+                    <p class="info__area--text">
+                        Welcome to the comproDLS<span>TM</span> OnBoarding 
+                        Application for mentors and freshers. Please sign in using your organization email and password.
+                        <br><br>
+                        In case you do not have an account, click on 'Sing Up now' to get started.
+                    </p>
+                </div>
+            </div>
+            <div class="container__right">
+                <div class="login__area">
+                    <p class="signup web-visi">
+                        Not a member? &nbsp;&nbsp;<span><a href="#">
+                            Request an Account</a>
+                        </span>
+                    </p>
+    
+                    <div class="login__box">
+
+                        <div class="msg__box" id="message_box">
+                            <p class="msg__box--message" id="message">
+                            </p>
+                        </div>
+
+                        <p class="sign-in">
+                            Sign in
+                        </p>
+                        <div class="input__area">
+                            <form class="login__form">
+                                <input class="login__form--input-field" type="text" id="email" placeholder="Email" name="e_mail">
+                                
+                                <input class="login__form--input-field" type="password" id="password" placeholder="Password" name="pass">
+    
+                                <p class="login__form--forgot_pass">
```

Comment:
class `login__form--forgot_pass` suggests that this is `forgot_pass` variant of `login__form` which is not correct here. Similar issues at other instances as well.